src/hooks/useTxDetails/useTxDetails.ts

```diff
@@ -54,6 +54,14 @@ export const getSellTx = (tx: Tx) => tx.transfers.find(t => t.type === chainAdap
 export const isSupportedContract = (tx: Tx) =>
   tx.data?.method ? SUPPORTED_CONTRACT_METHODS.has(tx.data?.method) : false
 
+export const isTradeContract = (
+  buyTx: chainAdapters.TxTransfer,
```

Comment:
would rename these variables to buyTransfer and sellTransfer, as transfers are components of a tx

Reply:
Yes, I can do that. I used buyTx and sellTx since that was the conventions already established in the file from the `getSellTx` and `getBuyTx` functions.

Reply:
Yeah, we should fix that - there are a number of places in `web` where we incorrectly refer to transfers as transactions.

I think it's due to the `TxType` from `unchained`'s `ChainAdapters` being misused - for whatever reason it's used as the type of `type` on a `TxTransfer`, which is incorrect.

Reply:
@0xApotheosis I actually started down the route of fixing it in the yearn issue but there was a ton of references so I noped out of it and stuck with Tx; left it to be done for the transactions refactor.

Reply:
Yep, fair enough too 😅

Reply:
@0xdef1cafe lets get this resolved